app/unit-tests/src/org/commcare/CommCareTestApplication.java

```diff
@@ -282,6 +284,7 @@ public void prepareTest(Object test) {
 
     @Override
     public ModernHttpRequester buildHttpRequester(Context context, String url, Map<String, String> params,
+                                                  Multimap<String, String> multiParams,
```

Comment:
why do we need both 'params' and 'multiParams'?

Reply:
I was earlier trying to limit the changes but it does look like a cleaner API with one params - https://github.com/dimagi/commcare-android/pull/2584/commits/606ba82f290e818b90e9a10f4cfca5dfa87b796a and   https://github.com/dimagi/commcare-core/pull/1151